var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -254,86 +253,9 @@ def _microarch_target_args(self):
 
         return args
 
-    @property
```

Comment:
I'm guessing this was for ease of defining a Windows build. Most of this logic will have to be reintroduced conditionally. We can chat separately about how to do this but for now (pending https://github.com/spack/seps/pull/3) you generally have to inherit from `Package` (vs. `CMakePackage`) if the build system differs in different cases.